sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f

```diff
@@ -237,7 +237,7 @@ SUBROUTINE INITPOST_GFS_NEMS_MPIIO(iostatusAER)
         end do
       end do
 
-!     write(*,*)'nrec=',nrec
+!     write(0,*)'nrec=',nrec
```

Comment:
@lipan-NOAA Please revert this change. We would like the no-fatal messages go to stdout instead of stderr.

Reply:
reverted it